src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1677,7 +1677,7 @@
               Grid.Row="2"
               Grid.Column="4"
               Grid.RowSpan="2">
-            <TabControl Name="tabDynamic" ItemsSource="{Binding}">
+            <TabControl Name="tabDynamic" Background="#353535" BorderThickness="0" ItemsSource="{Binding}">
```

Comment:
This would remove the white border around the extensions side bar.

Reply:
Thanks!